python/didcommgrpc.py

```diff
@@ -0,0 +1,136 @@
+from typing import Any
```

Comment:
This file should probably be renamed to `okapi.py` or something similar, as we don't use the `didcommgrpc` term anymore.

Reply:
Fixed that and the demo file